Description: Add unistd.h
 For a rationale see http://gcc.gnu.org/gcc-4.7/porting_to.html at section 
 "Header dependency changes".
Author: Michael Fladischer <Fladischer.Michael@fladi.at>
Last-Update: 2012-04-04
Forwarded: https://bugs.launchpad.net/libmemcached/+bug/973287

--- a/util/pidfile.cc
+++ b/util/pidfile.cc
@@ -48,6 +48,7 @@
 #include <sstream>
 #include <sys/stat.h>
 #include <sys/types.h>
+#include <unistd.h>
 
 extern "C" {
 
